<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfWindow</class>
 <widget class="QMainWindow" name="ConfWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EQda</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="AppBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>680</width>
      <height>695</height>
     </rect>
    </property>
    <property name="title">
     <string>Target APP</string>
    </property>
    <widget class="QPushButton" name="LoadAppListBtn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load List</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PackageName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>410</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Selected APP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PackageNameBtn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="AppList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>660</width>
       <height>635</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>PID</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>APP Name</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Package(Identifier)</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DeviceBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>680</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Target Device</string>
    </property>
    <widget class="QTableWidget" name="DeviceList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>580</width>
       <height>80</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TYPE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NAME</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="searchBtn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SelDeviceBtn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>50</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="DeviceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>105</y>
       <width>660</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Selected Device: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>125</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>iOS List</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="iOSList">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>125</y>
       <width>490</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="iOSListBtn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>125</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>165</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Script Edit Program</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScriptEditProgram">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>165</y>
      <width>470</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ScriptEditSaveBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>165</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="analysisMenu"/>
   <addaction name="separator"/>
   <addaction name="configMenu"/>
   <addaction name="separator"/>
  </widget>
  <action name="analysisMenu">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Analysis</string>
   </property>
  </action>
  <action name="configMenu">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="SelAttachDevice">
   <property name="text">
    <string>Device 변경</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
